plugin/al/lib/AL_USDMaya/AL/usdmaya/PluginRegister.h

```diff
@@ -171,7 +171,7 @@ MStatus registerPlugin(AFnPlugin& plugin)
 {
   GlfGlewInit();
 
-  #if (PXR_MAJOR_VERSION > 0) || (PXR_MINOR_VERSION >= 19 && PXR_PATCH_VERSION >= 5) 
+  #if USD_VERSION_NUM >= 1903
```

Comment:
Similar here, 1905?

Reply:
This fix was originally included in the pr/usd-0.19.5-update branch, but it
can actually be applied in 19.03. I think it only started creating errors
in 19.05, but it's possible it was simply mis-categorized. In any case,
including these changes in 19.03 should work, and is probably "best".